pkgs/development/python-modules/pytest-flask/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest, flask, setuptools_scm }:
+
+buildPythonPackage rec {
+  pname = "pytest-flask";
+  version = "0.15.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0jdzrib94vwfpl8524h34aqzqndh3h4xn706v32xh412c8dphx6q";
+  };
+
+  doCheck = true;
+
+  propagatedBuildInputs = [
+    pytest
+    flask
```

Comment:
I'd think you we're missing a requirement https://github.com/pytest-dev/pytest-flask/blob/0.15.0/requirements/main.txt#L3

Reply:
`Werkzeug` is already a (propagated) dependency of flask though... is it customary to explicitly refer to such implied dependencies?

Reply:
Yes, all dependencies that are needed should be explicitly added. Especially if they're in the `requriements*`.

Reply:
Explicitly mentioned dependencies should be added. Transitive dependencies that are not explicitly added should not be added, because they will be automatically obtained.

The reason why we should include dependencies despite them already being brought in through other dependencies is because the code may explicitly use that library.

Reply:
Clear enough :-)

Done in the latest push